torch/_inductor/scheduler.py

```diff
@@ -969,10 +988,8 @@ def __init__(
         else:
             self.scheduler = scheduler
             self.snodes = nodes
-            self.node = None
-
-            self.node = None
-            self.users = None
+            self.node = None  # type: ignore[assignment]
+            self.users = None  # type: ignore[assignment]
```

Comment:
I think `self.users` is an Optional type, so this `type: ignore` shouldn't be necessary

Reply:
`super().__init__()` is not called so it needs to be re-declared. Anyway added type hints.